### DISCUSSION TITLE
How do I draw a sprite directly on the camera?

### DISCUSSION THREAD

**Discussion by bogdan-ov** [HaxeFlixel/flixel#2857] [Q&A]
I want to draw a sprite from bitmap directly on the camera. It might look something like this: (creating a separate sprite and then rendering it using the `.draw()` isn't for me)
```haxe
// ...

class PlayState extends FlxState {
  var mySprite:FlxSprite;  

  override public function create() {
    super.create();

    mySprite = new FlxSprite();
    mySprite.loadGraphic("path/to/sprite.png");

    FlxG.bitmap.add("path/to/sprite.png", false, "my-key"); // Cache sprite bitmap
  }

  override function draw() {
    super.draw();

    mySprite.draw(); // This is not what I need

    var posX = 100;
    var posY = 40;
    var cache = FlxG.bitmap.get("my-key");
    if (cache != null)
      FlxG.camera.drawSpriteFromBitmapData(cache.bitmap, posX, posY); // Of course there is no such method
  }
}
```

Hot can I do this?

**Thanks in advance!**

**Comment by Geokureli** [comment]
Currently FlxSprites are already "drawn directly to the camera". Why not just create a FlxSprite? Take a step back and tell us what problem you are trying to solve with this method, or what problem you're facing by using the existing method

**Reply by bogdan-ov** [reply]
Sorry, but I've already found a solution (I used `sprite.draw()`)

**Reply by Geokureli** [reply]
lol
<img width="353" alt="Screenshot 2023-07-21 at 9 20 35 AM" src="https://github.com/HaxeFlixel/flixel/assets/2609513/7b9e8a8c-3090-4f5c-acb4-474ac96c12f2">

**Reply by bogdan-ov** [reply]
Hahaha. Yes, I'm a fool
